src/components/atoms/date-picker/custom-header.tsx

```diff
@@ -1,34 +1,37 @@
-import moment from "moment"
 import React from "react"
-import "react-datepicker/dist/react-datepicker.css"
 import NativeSelect from "../../molecules/native-select"
 import { ReactDatePickerCustomHeaderProps } from "react-datepicker"
-import { months, years } from "./utils"
+import { monthNames, getYearRange } from "./utils"
 
 const CustomHeader = ({
   date,
   changeYear,
   changeMonth,
 }: ReactDatePickerCustomHeaderProps) => {
+  const month = date.toLocaleString("default", { month: "long" })
```

Comment:
todo: fix the locale so that it won't depend on the system locale. Thinking that if this runs on a computer with e.g. da-DK locale you would get "Juni" and not "June" which would break the implementation. Maybe we could use `getUTCMonth` and make the implementation depend on the numerical value of the month instead of the name?

Reply:
let me know if the adjustment makes sense. I just used the `getMonth()` method which returns the numerical month (0-12) and use the `monthNames` map to get the month name in english